.papr.inventory

```diff
@@ -15,6 +15,17 @@ osm_cluster_network_cidr=10.128.0.0/14
 openshift_portal_net=172.30.0.0/16
 osm_host_subnet_length=9
 
+[all:vars]
+# bootstrap configs
+openshift_node_groups='[{"name":"node-config-master","labels":["node-role.kubernetes.io/master=true","node-role.kubernetes.io/infra=true"]},{"name":"node-config-node","labels":["node-role.kubernetes.io/compute=true"]}]'
+openshift_master_bootstrap_enabled=true
+openshift_master_bootstrap_auto_approve=true
+openshift_master_bootstrap_auto_approver_node_selector='{"region":"infra"}'
```

Comment:
This looks like a json string; if you want a dictionary, drop the single quotes.

Reply:
It's supposed to be a JSON string IIRC

Reply:
Nope, it isn't. Fixing